### ISSUE TITLE
Full E2E run with functional embeddings to compare results to original executions (notebook based)

### ISSUE THREAD

**Issue by pascalwhoop** [everycure-org/matrix#118]
**Describe your hypothesis.**

Running the pipeline in a cluster setup outperforms single machine executions and allows
us to run many pipelines in parallel which will speed up team performance over the long
run

**Describe your experiment.**

- Build a cloud cluster based deployment of our pipeline
- Execute the pipeline on the cluster
- Verify the results are near equivalent to the notebook based implementations but speed
    of computation and reproducability is improved

**My experiment is successful when**

- All hyperparameters are the same as the PoC implementations and the model results are nearly
    the same as well

# Open Work before we can benchmark results

- [x] #108 
- [x] First run merge: #39 
- [x] Fix embeddings #97 
- [x] #102 
- [x] #98
- [x] #101 

# Previous work already done

Below is an image that showed the work needed to be done to create the first E2E run which worked from a scaling perspective but had faulty embeddings. 

<img width="2122" alt="image" src="https://github.com/user-attachments/assets/dc65dffd-d91f-40ec-956a-b6b2422c3e58">
